app/api/views/questions.py

```diff
@@ -0,0 +1,78 @@
+from flask import request, jsonify
+from flask_login import login_required, current_user
+from app.api import api
+from app.api.business import questions_business
+from app.api.helpers import (
+    abort,
+    forbidden,
+    get_email_domain,
+    role_required,
+    permissions_required
+)
+from app.api.business.errors import (
+    NotFoundError,
+    ValidationError
+)
+from ...utils import get_json_from_request
+
+
+@api.route('/brief/<int:brief_id>/questions', methods=['GET'])
+@login_required
+@role_required('buyer')
+def get_questions(brief_id):
+    result = None
+    try:
+        result = questions_business.get_questions(brief_id)
```

Comment:
Currently this return data includes supplier name and supplier code for questions asked by a supplier - do we want this available for any buyer to see against any other brief? maybe strip out `result['supplierName']` and `result['supplier_code']` for buyers who don't have privs for this brief id?

Reply:
Actually, should this entire endpoint be restricted to only those who privileges against this brief id? the `/answers` endpoint includes the questions and seems more inline with what we currently consider public info regarding questions and answers.

Reply:
fixed